src/generic/dev-server/index.ts

```diff
@@ -0,0 +1,53 @@
+
+import {BuilderContext, BuilderConfiguration, BuildEvent, Builder} from '@angular-devkit/architect';
+import {DevServerBuilder, DevServerBuilderOptions, BrowserBuilderSchema, BrowserBuilder, NormalizedBrowserBuilderSchema } from '@angular-devkit/build-angular';
+import {Path, virtualFs} from '@angular-devkit/core';
+import {Observable} from 'rxjs';
+import {tap, switchMap} from 'rxjs/operators';
+import * as fs from 'fs';
```

Comment:
Please remove unused imports

Reply:
fs.stat is used as part of the method signature for the buildWebpackConfig method.... Is there a better way to to import just the stat method from the fs module?

Reply:
Yep, like this:
`import {Stats} from 'fs';` and then in sig: ` host: virtualFs.Host<Stats>,`